mappings/net/minecraft/entity/ai/goal/ActiveTargetGoal.mapping

```diff
@@ -1,4 +1,4 @@
-CLASS net/minecraft/unmapped/C_wnzeqexr net/minecraft/entity/ai/goal/FollowTargetGoal
+CLASS net/minecraft/unmapped/C_wnzeqexr net/minecraft/entity/ai/goal/ActiveTargetGoal
```

Comment:
This sounds really strange, needs comparison with mojmap.

Reply:
Odd... Both are weird. Is there a `TargetGoal` already?

Reply:
Some inner classes are named `TargetGoal`, but there isn't any upper level class that's named just that

Reply:
Then imo I would name it `TargetGoal`, the `Active` prefix doesn't seem useful, or rather doesn't make much sense in the context of goals.